dotnet-desktop-guide/framework/wpf/advanced/printing-overview.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Printing Overview"
+description: Learn about the printing and print system management APIs that Microsoft .NET Framework offers application developers using Windows Presentation Foundation (WPF).
```

Comment:
Slightly over character minimum (160 characters). Consider: "Learn about the printing and print system management APIs that the .NET Framework offers application developers using Windows Presentation Foundation (WPF)."

Reply:
Change made.